cypress/models/registerModule.js

```diff
@@ -0,0 +1,96 @@
+/// <reference types = "Cypress" />
+import data from "../fixtures/data.json";
+import sidebar from "./sidebarModule";
+import navigation from "./navigationModule";
+import faker from "faker";
+
+let fakerUser = {
```

Comment:
You can freely use `const` here too:

```suggestion
const fakerUser = {
```